modular_zubbers/code/modules/vending/wardrobe.dm

```diff
@@ -0,0 +1,10 @@
+/obj/machinery/vending/wardrobe/sec_wardrobe/Initialize(mapload)
+	products += list(
+		/obj/item/clothing/head/helmet/metrocophelmet = 5,
+		/obj/item/clothing/suit/armor/metrocop = 5,
+	)
+	premium += list(
+		/obj/item/clothing/suit/armor/metrocopriot = 3,
```

Comment:
```suggestion
		/obj/item/clothing/suit/armor/metrocopriot = 1,
```
I'm a little iffy on adding riot gear to station vendors since it's a cargo orderable, and these vendors are all over the station, so previously a cargo-only or armoury-only item is now able for anyone to buy after tiding into a sec checkpoint.
Put the supply to 1 to satiate my moaning and I'll merge it.

Reply:
Honestly didn't know  about the riot stuff and was just going off the other Premium item's amount. Its now changed to 1 instead of 3.